src/renderer/features/governance/components/ReferendumDetails/ReferendumDetailsDialog.tsx

```diff
@@ -1,30 +1,30 @@
 import { useGate, useStoreMap } from 'effector-react';
 
-import { type Chain, type Referendum } from '@shared/core';
+import { type Chain } from '@shared/core';
 import { pickNestedValue } from '@shared/lib/utils';
 import { useI18n } from '@app/providers';
 import { BaseModal, Plate } from '@shared/ui';
 import { useModalClose } from '@shared/lib/hooks';
-import { approveThresholdModel, supportThresholdModel } from '@entities/governance';
 import { detailsAggregate } from '../../aggregates/details';
 import { ProposalDescription } from './ProposalDescription';
 import { VotingStatus } from './VotingStatus';
 import { DetailsCard } from './DetailsCard';
+import { AggregatedReferendum } from '../../types/structs';
 
 type Props = {
   chain: Chain;
-  referendum: Referendum;
+  referendum: AggregatedReferendum;
```

Comment:
Maybe call it in another way and extract `referendum` to exclude using of `referendum.referendum`?

Reply:
Merged raferendum with additional fields directly
```ts
referendum & {
 /* ... */
}
```